lib-font.js

```diff
@@ -3,185 +3,199 @@ import { Event, EventManager } from "./src/eventing.js";
 import { SFNT, WOFF, WOFF2 } from "./src/opentype/index.js";
 import { loadTableClasses } from "./src/opentype/tables/createTable.js";
 import { setupFontFace } from "./src/utils/fontface.js";
-import { validFontFormat } from  "./src/utils/validator.js";
+import { validFontFormat } from "./src/utils/validator.js";
```

Comment:
ignore this file. It's just `prettier` finally being told to format this file =D